src/parser/classes/livechat/items/LiveChatPaidMessage.ts

```diff
@@ -20,6 +20,10 @@ class LiveChatPaidMessage extends YTNode {
     is_verified_artist: boolean | null;
   };
 
+  headerBackgroundColor: number;
+  headerTextColor: number;
+  bodyBackgroundColor: number;
+  bodyTextColor: number;
```

Comment:
Note that these, like the others, must be in snake_case instead of camelCase. 
Other than that, everything looks great.

Thanks for contributing! <3

Reply:
Confirmed.
I fixed it.